docs/devsetup/cpp/index.md

```diff
@@ -70,7 +70,7 @@ For example:
         --disable-release Linux --prefix=/home/*your_user_name*/icu/mine/inst >
         config.out 2>&1`
     *   build: make -j5 check > out.txt 2>&1
-    *   Be sure to test with gcc and g++ too! `CC=gcc CXX=g++
+    *   Be sure to test with both gcc/g++ and clang/clang++! `CC=clang CXX=clang++
```

Comment:
```suggestion
    *   Be sure to test with both gcc/g++ and clang/clang++! `CC=clang CXX=clang++`
```

Reply:
Maybe suggest using `runConfigureICU Linux/clang` instead?

Reply:
Better, but the quoting is still off, now there's no initial ` before the code block. Please fix that and then squash all the manual changes together into one single commit (after which then follows the commit that I made with the autogenerated changes).

Reply:
Done.

Reply:
Looks good, but please squash all manual changes, including the one's that I've made, into a single commit, the only thing that's advantageous to keep separate are the autogenerated changes.

Reply:
Okay, I think you must be talking about the commit `ICU-22556 Update configure files from configure.ac using autoreconf.` that should be separate, right? If that's the case, the branch should be good now.